RT1060/tags

```diff
@@ -0,0 +1,868 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
I think tags file was accidentally added. Maybe worth adding to .gitignore?

Reply:
oops!